src/components/AuthForm/AuthForm.css

```diff
@@ -0,0 +1,35 @@
+form {
+  background-color: #ff8513;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+  gap: 30px;
+  margin: 20px;
+  border: #ff00aa solid 13px;
+  padding: 20px;
+  border-radius: 20px;
+}
+
+input[type='submit'] {
+  border: rgb(179, 255, 0) 5px solid;
+  padding: 10px;
+  border-radius: 10px;
+  background-color: aqua;
+}
+
+input[type='submit']:hover {
+  cursor: pointer;
+}
+
+form p {
+  margin: -30px 0px 0px -20px;
```

Comment:
There are hardly any good use cases for negative margins and I encourage you to reimagine this layout to not use them. Remember that margins "collapse" when stacked on top of each other (outside of flex) and empty elements will still take up space in flex (gap and all).